lib/ui/views/new_dish/new_dish_view.dart

```diff
@@ -117,16 +119,39 @@ class NewDishView extends StackedView<NewDishViewModel> with $NewDishView {
                   ),
                 ),
                 SizedBox(
-                  height: 80.h,
+                  height: 20.h,
+                ),
+                ElevatedButton(
+                  onPressed: () => viewModel.selectImage(),
```

Comment:
you can do a function tear down since you are not passing any args to the function